integrationTests/cypress/integration/page.feature

```diff
@@ -0,0 +1,4 @@
+Feature: "A Smeagol Page displays its content, has functions for manipulation of the content and changeing the branch"
```

Comment:
It seems like this feature doesn't test anything. There is no `Then`

Reply:
This is only a "stub", as integration test coverage will be implemented in another sprint.